lib/go-tc/invalidationjobs.go

```diff
@@ -272,9 +275,83 @@ func (job *InvalidationJobInput) Validate(tx *sql.Tx) error {
 	if len(errs) > 0 {
 		return errors.New(strings.Join(errs, ", "))
 	}
+
+	errs = job.ValidateUniqueness(tx, dsID)
+	if len(errs) > 0 {
+		return errors.New(strings.Join(errs, ", "))
```

Comment:
`errs` is a misnomer, it's actually a `[]string`. I'll change it so that the naming makes more sense (and it will just actually return an error instead of a string).

Reply:
https://godoc.org/github.com/apache/trafficcontrol/lib/go-util#JoinErrs might be useful